department-of-veterans-affairs/caseflow
# Create Tasks and Appeals Page

Resolves [Intake Form Step 2.1: Review Tasks & Appeals - Mail Tasks](https://jira.devops.va.gov/browse/APPEALS-29414)

# Description
Creates UI for setting tasks during the intake flow.

## Acceptance Criteria

## Testing Plan
1. Visit http://localhost:3000/queue/correspondence/test/intake
2. Click to go to step 2.
3. Verify the UI matches the Jira ticket.

# Frontend
## User Facing Changes
<img width="2405" alt="Screenshot 2023-10-30 at 12 18 47" src="https://github.com/department-of-veterans-affairs/caseflow/assets/2421172/7d93ae32-6e76-47ab-becc-146a7334aae8">
